main.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-tools/go-steputils/input"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+)
+
+// Config ...
+type Config struct {
+	IpaPath           string          `env:"ipa_path"`
+	PkgPath           string          `env:"pkg_path"`
+	ItunesConnectUser stepconf.Secret `env:"itunescon_user,required"`
+	Password          stepconf.Secret `env:"password"`
```

Comment:
is not password required?

Reply:
Nope, because if the user use AppPassword it will use that instead of password.

-> 
```
if err := input.ValidateIfNotEmpty(string(cfg.Password)); err != nil {
		if err := input.ValidateIfNotEmpty(string(cfg.AppPassword)); err != nil {
			return fmt.Errorf("Neither password nor app_password is provided")
		}
	}
```